Sparse CUDA: rename files *.cu -> *.cpp

IvanYashchuk opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #63948 Sparse CSR CUDA: add `torch.add` with all inputs sparse
* #63511 Sparse CSR CUDA: add `torch.addmm` with all inputs sparse
* #63510 Sparse CSR: Add tensor.resize_ and tensor.copy_
* #63509 Sparse CSR: Add torch.empty
* #62180 Sparse CSR CPU: add `triangular_solve_out`
* #61858 Sparse CSR CUDA: add `triangular_solve_out`
* #61857 Port triangular_solve to structured kernel
* #61536 Sparse CSR CPU: add `addmv_out`
* #61408 Sparse CSR: add test parametrization for enabling sparse_csr
* #61407 Sparse CSR CUDA: add `addmv_out`
* #60838 Add cuSPARSE descriptors and update CSR addmm
* **#63894 Sparse CUDA: rename files *.cu -> *.cpp**

This PR introduces a few code structure changes. There is no need to use
.cu extension for pure c++ code without cuda. Moved
`s_addmm_out_csr_sparse_dense_cuda_worker` to a separate cpp file from
cu file.

cc @nikitaved @pearu @cpuhrsch @IvanYashchuk @ngimel

Differential Revision: [D30548771](https://our.internmc.facebook.com/intern/diff/D30548771)

IvanYashchuk committed:
Sparse CUDA: rename files *.cu -> *.cpp

This PR introduces a few code structure changes. There is no need to use
.cu extension for pure c++ code without cuda. Moved
`s_addmm_out_csr_sparse_dense_cuda_worker` to a separate cpp file from
cu file.

[ghstack-poisoned]

IvanYashchuk committed:
Update on "Sparse CUDA: rename files *.cu -> *.cpp"


This PR introduces a few code structure changes. There is no need to use
.cu extension for pure c++ code without cuda. Moved
`s_addmm_out_csr_sparse_dense_cuda_worker` to a separate cpp file from
cu file.

cc @nikitaved @pearu @cpuhrsch @IvanYashchuk @ngimel

[ghstack-poisoned]

IvanYashchuk committed:
Update on "Sparse CUDA: rename files *.cu -> *.cpp"


This PR introduces a few code structure changes. There is no need to use
.cu extension for pure c++ code without cuda. Moved
`s_addmm_out_csr_sparse_dense_cuda_worker` to a separate cpp file from
cu file.

cc @nikitaved @pearu @cpuhrsch @IvanYashchuk @ngimel

[ghstack-poisoned]

IvanYashchuk committed:
Update on "Sparse CUDA: rename files *.cu -> *.cpp"


This PR introduces a few code structure changes. There is no need to use
.cu extension for pure c++ code without cuda. Moved
`s_addmm_out_csr_sparse_dense_cuda_worker` to a separate cpp file from
cu file.

cc @nikitaved @pearu @cpuhrsch @IvanYashchuk @ngimel

[ghstack-poisoned]

IvanYashchuk committed:
Update on "Sparse CUDA: rename files *.cu -> *.cpp"


This PR introduces a few code structure changes. There is no need to use
.cu extension for pure c++ code without cuda. Moved
`s_addmm_out_csr_sparse_dense_cuda_worker` to a separate cpp file from
cu file.

cc @nikitaved @pearu @cpuhrsch @IvanYashchuk @ngimel

[ghstack-poisoned]

cpuhrsch reviewed (approved):
Please address the comment, will merge after.

IvanYashchuk committed:
Update on "Sparse CUDA: rename files *.cu -> *.cpp"


This PR introduces a few code structure changes. There is no need to use
.cu extension for pure c++ code without cuda. Moved
`s_addmm_out_csr_sparse_dense_cuda_worker` to a separate cpp file from
cu file.

cc @nikitaved @pearu @cpuhrsch @IvanYashchuk @ngimel

Differential Revision: [D30548771](https://our.internmc.facebook.com/intern/diff/D30548771)

[ghstack-poisoned]

IvanYashchuk committed:
Update on "Sparse CUDA: rename files *.cu -> *.cpp"


This PR introduces a few code structure changes. There is no need to use
.cu extension for pure c++ code without cuda. Moved
`s_addmm_out_csr_sparse_dense_cuda_worker` to a separate cpp file from
cu file.

cc @nikitaved @pearu @cpuhrsch @IvanYashchuk @ngimel

Differential Revision: [D30548771](https://our.internmc.facebook.com/intern/diff/D30548771)

[ghstack-poisoned]
